libs/datasets/timeseries.py

```diff
@@ -741,6 +741,7 @@ def print_stats(self, name: str):
                 textwrap.indent(by_level.to_string(), "  "),
             ]
         )
+        print(datetime.datetime.now())
```

Comment:
I'm slowly trying to understand why our actions are so slow and adding more logging to help understand where the time goes...